packages/vapor/scss/components/tab.scss

```
@@ -21,12 +23,14 @@
         transition: all 0.2s ease;
 
         &.active {
-            color: var(--tab-text-color);
+            color: var(--color);
             border-bottom: $tab-bottom-border-height solid var(--tab-border-color);
         }
 
         &.disabled {
-            color: var(--medium-grey);
+            --color: var(--medium-grey);
+
+            color: var(--color);
```

You should only override the variable, no need to redefine the same rule that is already in .tab
```suggestion
            --color: var(--medium-grey);
```